### PR TITLE
Create wicked-bendathlon.md

### PR DESCRIPTION
**Pull request by esparkhu**
Rules for Wicked Bender

### CODE REVIEW

File: content/events/wicked-bendathalon.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@title 1v1 Wicked Bender
+
+# 1v1 Wicked Bender (aka Glory Wicked-Bender, aka Wicked Bendathalon)
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]: We should decide on one thing to call it, this is too long for a title

File: content/events/wicked-bendathalon.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+A pair of opponents play a set of two 1v1 games, both on Tiny maps in Glory mode. Players take turns playing matching tribes. Each player hosts one of the games, usually with the weaker tribe. Cumulative total is decided by sets per team similar points to team games.  Winner will be decided by score leaderboard or finals round play off in the case of more than one player getting above 20k (two wins).
+
+## The rules
+
+### Acceptable Tribes ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]: Don't match the hashes on the right, you only need them on the left
```suggestion
### Acceptable Tribes
```

File: content/events/wicked-bendathalon.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Imperius, Xin-xi, Oumaji, Zebasi, Hoodrick, Quetzali, Aimo, and Yaddak
+
+### Map settings ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Map settings
```

File: content/events/wicked-bendathalon.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Tiny map size only. Acceptable map types are Continents, Lakes, Drylands and Archipelago and is chosen at players discretion. Default is Continents.
+
+### Battleships banned ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Battleships banned
```

File: content/events/wicked-bendathalon.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Battleships are banned. If you accidentally create one or get one from a ruins, you must land it within your own territory and may not use it to explore. Failure to disband the battleship, using it to fire on an enemy or scouting with it will incur a penalty.
+
+### Giants ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Giants
```

File: content/events/wicked-bendathalon.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+Giants can only be used defensively. They cannot move or attack into enemy territory, siege/conquer cities or be made into ships. You may make a giant into a boat, but it is still restricted to only moving and attacking in your territory and no-mans-land. If a border expands over your giant, you have to immediately move it out of enemy territory or disband it, though you will not be penalized.
+
+### Penalties ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Penalties
```

File: content/events/wicked-bendathalon.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+### Battleships banned ###
+Battleships are banned. If you accidentally create one or get one from a ruins, you must land it within your own territory and may not use it to explore. Failure to disband the battleship, using it to fire on an enemy or scouting with it will incur a penalty.
+
+### Giants ###
+Giants can only be used defensively. They cannot move or attack into enemy territory, siege/conquer cities or be made into ships. You may make a giant into a boat, but it is still restricted to only moving and attacking in your territory and no-mans-land. If a border expands over your giant, you have to immediately move it out of enemy territory or disband it, though you will not be penalized.
+
+### Penalties ### 
+Breaking the limitations on battleships or giants is penalized with 1000 points per unit and turn. Penalties can drop your point score below the minimum 3000.
+
+## Restart Rules ##
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
## Restart Rules
```

File: content/events/wicked-bendathalon.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+There are three types of restarts.
+
+### Proximity ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Proximity
```

File: content/events/wicked-bendathalon.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+### Penalties ### 
+Breaking the limitations on battleships or giants is penalized with 1000 points per unit and turn. Penalties can drop your point score below the minimum 3000.
+
+## Restart Rules ##
+There are three types of restarts.
+
+### Proximity ### 
+You and your opponent have only one tile between the border of your capitals.
+
+### Isolation ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Isolation
```

File: content/events/wicked-bendathalon.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+## Restart Rules ##
+There are three types of restarts.
+
+### Proximity ### 
+You and your opponent have only one tile between the border of your capitals.
+
+### Isolation ### 
+You need to buy a tech to reach a second village (typically sailing or climbing) or the nearest village without a tech buy is five or more squares away from your capital. Single tiles along edges don’t have to be scouted to claim isolation restart. However, exploring these tiles and revealing a village in range will forfeit the restart option.
+
+### Barren ###
```
Comment (Artemis21) [inline content/events/wicked-bendathalon.md]:
```suggestion
### Barren
```

**Review by Artemis21 — CHANGES_REQUESTED** [review]
Formatting issues

File: content/events/wicked-bendathalon.md (hunk 1 above)
Comment (esparkhu) [reply content/events/wicked-bendathalon.md]: OKay. Let's do Wicked Bendathlon. That is what the event commissioners have agreed on.

**Review by esparkhu — COMMENTED** [review]
I changed the name of the file. This will mean updating the links in the events page

**Pull request merged**